src/seedu/addressbook/commands/EditNameCommand.java

```diff
@@ -0,0 +1,64 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.Person;
+
+public class EditNameCommand extends Command{
```

Comment:
Missing header comments. Every class should have header comments as specified in the coding style.